packages/cells/src/model.ts

```diff
@@ -301,6 +301,19 @@ export class CellModel extends CodeEditor.Model implements ICellModel {
     /* no-op */
   }
 
+  public switchSharedModel(
```

Comment:
Could you add docstring please? 

> I suggest removing `public` as we have the convention not to write it explicitly in JLab.

```suggestion
  switchSharedModel(
```